README.rst

```diff
@@ -260,6 +260,23 @@ existing ones. These are know to work well with Volto layout and grid system::
 sure your add-on's profiles are applied AFTER this one.**
 
 
+Slate support
+-------------
+
+Starting with Volto 16, the default richtext implementation uses SlateJS. We
+include 2 converters:
+
+- Volto's flavour of Slate JSON to HTML (``Slate2HTML``)
+- HTML to Volto's flavour of Slate JSON (``HTML2Slate``)
+
+These two classes can be inherited and extended for your custom tags and
+plugins.
+
+It's also possible to include an optional ``htmlblock.zcml`` configure file
+(located in ``plone.volto.slate`` ). The effect is that the value of the slate
+blocks will be stored as an HTML and it will be transparently converted to
+Slate, when the content is loaded in Volto.
```

Comment:
If we have perfect converters in/out, I think it's better to store html instead of custom JSON. But this deserves a discussion and a strategy on how to expose this to other Plone projects.

For the moment I'll add the note you suggest.